src/votekit/pref_profile.py

```diff
@@ -303,3 +300,15 @@ def format_as_float(percent_str):
         df.loc["Totals"] = sum_row  # type: ignore
 
         return df.fillna("")
+
+    def __add__(self, other):
+        """
+        Add two PreferenceProfiles by combining their ballot lists.
+        """
+        if isinstance(other, PreferenceProfile):
+            ballots = self.ballots+other.ballots
+            pp = PreferenceProfile(ballots=ballots)
+            pp.condense_ballots()
+            return pp
```

Comment:
Not sure what the current protocol is but this seems like something that could use some tests - easy to imagine places where the condensing leads to unintuitive results.